_includes/tutorials/anomaly-detection/ksql/markup/dev/create-suspicious-transactions-stream.adoc

```diff
@@ -0,0 +1,14 @@
+Using the table of suspicious names and stream of transactions, create a new stream of events containing transactions that were sent to an account name contained in the 'suspicious_names' table. We can do this by performing an `INNER JOIN`. In this case the `INNER JOIN` will couple events in the transaction stream where the "recipient" is the same as "company_name" in the suspicious_names table. The stream created below will continuously be populated by the coupled events created by the query.
```

Comment:
Consistency check: what should be in single quotes, backticks, double quotes?  Not sure we have an established style guide, WDYT?  cc: @bbejeck